### PR TITLE
Use graphql-client library

### PR DESCRIPTION
**Pull request by n1tranquilla**
fixes: #91, #107

### CODE REVIEW

File: src/transactions/functions.rs [hunk 1]
```diff
@@ -106,5 +148,153 @@ mod tests {
     }
 }
 
-
-
+fn build_query(public_key: Option<String>) -> transactions_query::TransactionQueryInput {
```
Comment (n1tranquilla) [inline src/transactions/functions.rs:151]: This is unfortunate. I tried following a builder pattern or trying 

```
transactions_query::TransactionQueryInput {
  // my fields
  ..Default::defaults()
}
```

But the model is not under my control and I couldn't seem apply the Default macro

Comment (trevorbernard) [reply src/transactions/functions.rs:120]: Can you omit them?

```
transactions_query::TransactionQueryInput {
  from: public_key,
  canonical: Some(true),
}
```

**Review by trevorbernard — APPROVED** [review]
:shipit:

File: src/transactions/functions.rs (hunk 1 above)
Comment (n1tranquilla) [reply src/transactions/functions.rs]: No, unfortunately. Struct not under my control

**Pull request merged**